library/rocksdbjni/RocksDbBuilder.kt

```diff
@@ -0,0 +1,36 @@
+package library.rocksdbjni
+
+import org.zeroturnaround.exec.ProcessExecutor
+import org.zeroturnaround.exec.ProcessResult
+import java.io.File
+import java.nio.file.Paths
+
+fun main() {
+    val baseDir = Paths.get(".")
+    val version = Paths.get("library").resolve("rocksdbjni").resolve("VERSION").toFile().useLines { it.firstOrNull() }
+    val javaHome = "/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home"
```

Comment:
There's no way to generify it huh? Do you have any idea if we can supply the argument from outside for this one, @vmax ?

Reply:
Arguments can't be supplied to a `genrule`. I raised an [issue](https://github.com/graknlabs/dependencies/issues/204) for someone to take a look at this in the future.